2018-12-05 Sinterklaas

```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+title: "Sinterklaas komt op ?5? december"
+date: 2018-11-12 12:00:00
+description: Sinterklaas bezoekt onze U6, U8 en U10-spelers!
+cover: /news/img/2018-12-05 Sinterklaas.jpg
+permalink: /news/2018-11-05 Sinterklaas/
+---
```

Comment:
best geen spaties gebruiken in cover & permalink, dit zal omgezet worden in http urls en daar zijn spaties niet handig voor mensen want dat wordt dan een %20 teken

Reply:
Ook in de bestandsnamen de spatie vervangen door een `-`

Reply:
bestandsnaam is ook onjuist, het zou `_content\news\_posts\2018-12-05-Sinterklaas.markdown` moeten zijn